assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-ideas-widget.scss

```diff
@@ -265,5 +265,9 @@
 
 	.googlesitekit-widget--ideaHubIdeas .googlesitekit-idea-hub__footer {
 		padding: 0;
+
+		.googlesitekit-idea-hub__footer--updated {
+			height: 36px;
```

Comment:
This 36px appears in multiple places

`assets/sass/components/idea-hub/_googlesitekit-idea-hub-pagination.scss`  sets a fixed height on the pagination buttons

`.googlesitekit-idea-hub__pagination--buttons  button.mdc-button`

And it's also used in `assets/sass/vendor/_mdc-button.scss` as a min-height for the button

I would expect this to be a Sass variable (if only for semantics). Happy to extract out!

Reply:
Good call—let's extract it out as a variable, then reference it here/elsewhere and note why we use it here (because with and without buttons we want the height of the container to be the same.)